tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -98,9 +98,21 @@ def _next(self) -> Optional[Dict[str, Any]]:
         return result
 
     _DEFAULT_LOCATIONS = {
-        "china": "china north 2",
-        "government": "usgovarizona",
-        "public": "westus2"
+        "AzureCloud": "westus2",
+        "AzureChinaCloud": "chinanorth2",
+        "AzureUSGovernment": "usgovarizona",
+    }
+
+    _MARKETPLACE_IMAGE_INFORMATION_LOCATIONS = {
```

Comment:
Currently these 2 properties are require by LISA for China and Government. They are working on making LISA generate those values (as it does for Public). In the meanwhile, any location will do.